server/src/main/java/org/jboss/pnc/buildagent/server/servlet/HttpInvoker.java

```diff
@@ -145,6 +152,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     private void handleOutput(CommandSession commandSession, int[] stdOut) {
         byte[] buffer = Arrays.charIntstoBytes(stdOut, StandardCharsets.UTF_8);
         stdoutChecksum.add(buffer);
+        logMatcher.append(Arrays.charIntsToString(stdOut));
```

Comment:
There is already an API to add different consumers of the stdOut. See sinkChannels in the BuildAgentServer.init().

Reply:
I've followed how the MD5 is done, it's also not done in `BuildAgentServer.init()`

Reply:
You could pass it via HttpInvokerFactory, but since md5 is there too you can also leave it as is.